### PR TITLE
store/tikv: tiny refactoring, change `twoPhaseCommitAction` to interface (#12845)

### PR DESCRIPTION
**Pull request by tiancaiamao**
Cherry-pick https://github.com/pingcap/tidb/pull/12845

### CODE REVIEW

**Comment by tiancaiamao** [comment]
PTAL @lysu @jackysp @cfzjywxk

File: store/tikv/2pc.go [hunk 1]
```diff
@@ -881,7 +884,6 @@ func (c *twoPhaseCommitter) commitSingleBatch(bo *Backoffer, batch batchKeys) er
 			SyncLog:  c.syncLog,
 		},
 	}
-	req.Context.Priority = c.priority
```
Comment (coocood) [inline store/tikv/2pc.go]: Why remove this line?

**Comment by coocood** [comment]
LGTM

File: store/tikv/2pc.go (hunk 1 above)
Comment (tiancaiamao) [reply store/tikv/2pc.go]: Because it's redundant with line 880

**Comment by cfzjywxk** [comment]
LGTM

**Pull request merged**